feature/matching-service/data/src/main/java/com/pob/umpa/feature/matching_service/data/qualifier/RealQualifier.kt

```
@@ -0,0 +1,9 @@
+package com.pob.umpa.feature.matching_service.data.qualifier
+
+import javax.inject.Qualifier
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class RealQualifier() {
+
+}
```

detekt (1.23.8)

[warning] The class or object RealQualifier is empty.

(detekt.empty-blocks.EmptyClassBlock)